packages/react-components/src/components/ChatMessage/ChatMessageComponentAsEditBox.tsx

```diff
@@ -98,7 +98,8 @@ export const ChatMessageComponentAsEditBox = (props: ChatMessageComponentAsEditB
 
   const onRenderFileUploads = useCallback(() => {
     return (
-      attachedFilesMetadata?.length && (
+      !!attachedFilesMetadata?.length &&
```

Comment:
should this be just a check for not undefined, as this would then allow the removal of the optional `?` in the length call?

Reply:
Pushed a change for this.